articles/azure-maps/how-to-use-map-control.md

```diff
@@ -21,9 +21,9 @@ This documentation uses the Azure Maps Web SDK, however the Azure Maps services
 
 To use the Map Control in a web page, you must have one of the following prerequisites:
 
-* [Make an Azure Maps account](quick-demo-map-app.md#create-an-azure-maps-account) and [obtain a primary subscription key](quick-demo-map-app.md#get-the-primary-key-for-your-account), also known as the primary key or the subscription key.
-
-* Obtain your Azure Active Directory (AAD) credentials with [authentication options](/javascript/api/azure-maps-control/atlas.authenticationoptions).
+1. An [Azure Maps account]
+1. A [subscription key]
+1. Obtain your Azure Active Directory (AAD) credentials with [authentication options]
```

Comment:
Because this list isn't sequential, it needs to be bulleted.

Reply:
I will change all of the articles prerequisite sections to be bulleted lists, note however these lists are sequential in the sense that you cannot get your subscription key until you have an Azure Maps account etc.